### PR TITLE
Add GraphCMS

### PR DESCRIPTION
**Pull request by ronakganatra**
Super clean site Pedro! Would love to feature GraphCMS here 🙇

### CODE REVIEW

**Review by peduarte — CHANGES_REQUESTED** [review]
Hey, thanks for the PR 🙏 

GraphCMS looks awesome. Let's get that added in!

One thing before we merge this in, can you place that after Kontent?

Trying to keep it in order of addition 😄

**Comment by ronakganatra** [comment]
Thanks @peduarte , changes done 😃 

Let me know whenever you're ready to try it out! Would love to help you get up to speed with GraphCMS ❤️ 
> GraphCMS looks awesome. Let's get that added in!

**Review by peduarte — APPROVED** [review]

**Pull request merged**
